Resources/public/js/views/actions/ez-translateactionview.js

```diff
@@ -41,7 +41,14 @@ YUI.add('ez-translateactionview', function (Y) {
          * @return Y.eZ.TranslateActionView the view itself
          */
         render: function () {
-            var container = this.get('container');
+            var container = this.get('container'),
+                translationsList = this.get('content').get('currentVersion').getTranslationsList(),
+                firstLinguageCodes = this._getFirstLanguageCodes(),
```

Comment:
I was not clear on my previous comment it seems, the typo is `firstLinguageCodes` instead of `firstLanguageCodes` :)